docs/registry/upload/_index.md

```diff
@@ -137,7 +137,7 @@ If you later wish to make your module public, you can use the [`viam module upda
     {
       "module_id": "acme:my-module",
       "visibility": "public",
-      "url": "https://github.com/acme-co-example/my-module",
+      "url": "https://github.com/{my-repo-name}/my-module",
```

Comment:
```suggestion
      "url": "https://github.com/<my-repo-name>/my-module",
```
maybe since we do it this way elsewhere?

Reply:
You're right, my brain was still writing Python. Thank you!